docs/central-entities.rst

```diff
@@ -233,6 +233,107 @@ You can see the full XLSForm `here <https://docs.google.com/spreadsheets/d/12oJZ
+
+Building a Form that updates an Entity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Updating Entities isn't too different from creating new Entities, but you'll have to specify the ``entity_id`` of the specific Entity instance you want to update in the ``entities`` sheet.
+
+.. rubric:: XLSForm
+
+.. csv-table:: entities
+  :header: list_name, entity_id
+ 
+  trees, ${tree}
+
+In this example, a tree in the ``trees`` Entity List will be updated when Submissions based on this Form are created. The particular tree to be updated must be given in the ``entity_id`` column, which here is an expression bound to the Submission that computes to the unique identifier of that tree for each data update.
+
+Once you have this update information set up, you can add ``save_to`` rules to your Form questions as with Entity creation:
+
+.. csv-table:: survey
+  :header: type, name, label, save_to
+ 
+  geopoint, current_location, Tree location, location
+  select_one species, species, Tree species
+  integer, circumference, Tree circumference in cm, circumference_cm
+  text, intake_notes, Intake notes
+
+Here, the ``current_location`` and ``circumference`` questions have been set up to update properties in the Entity, while the ``species`` and ``intake_notes`` questions don't update anything to the Entity.
+
+.. _central-entities_build-update-label:
+
+Updating the Entity Label
+"""""""""""""""""""""""""
+
+To update the label of an Entity from a Form, fill in the optional ``label`` column in the ``entities`` sheet as you would when setting the label on Entity creation.
+
+.. rubric:: XLSForm
+
+.. csv-table:: entities
+  :header: list_name, entity_id, label
+ 
+  trees, ${tree},"concat(${circumference}, ""cm "", ${species})"
+
+.. _central-entities_build-update-conditional:
+
+Set conditions under which an Entity is updated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can limit a Form to only update an Entity under certain conditions, based on an expression using the Entity and Submission data.
+
+.. rubric:: XLSForm
+
+.. csv-table:: entities
+  :header: list_name, entity_id, update
+ 
+  trees, ${tree}, true()
+
+In this case, ``true()`` is given in the optional ``update`` column, which means the Entity will always be updated upon submission. If an expression is given instead, the Entity is only updated when the expression evaluates to ``true`` or ``1``.
+
+.. _central-entities_build-update-create:
+
+Creating AND Updating Entities with one Form
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can give both a ``create`` and an ``update`` rule for the same Form. If only one of these expression evaluates to ``true`` or ``1``, then only that operation will be carried out. If both rules evaluate to ``true`` or ``1``, the Entity will be created if it does not exist (as identified by the ``entity_id`` expression), and updated if it does. Of course, if neither rule evaluates to ``true`` or ``1``, no Entity changes will occur.
```

Comment:
`create_if` and `update_if` (the XForms attributes are indeed `create` and `update`, for better or worse)